src/sickle.ts

```diff
@@ -224,35 +339,37 @@ class Annotator {
   }
 
   private visitProperty(className: string, p: ts.PropertyDeclaration | ts.ParameterDeclaration) {
-    let existingAnnotation = this.existingClosureAnnotation(p);
-    if (existingAnnotation) {
-      existingAnnotation += '\n';
+    let jsDoc = this.getJSDoc(p);
+    let existingAnnotation = '';
+    if (jsDoc) {
+      for (let { tagName, text } of jsDoc.tags) {
+        if (tagName === 'type') {
+          this.fail('@type not allowed in property declarations');
```

Comment:
We should prioritize bringing proper error reporting online (multiple errors, with locations) now that we add features like this.